packages/logger/src/writers/pino.ts

```diff
@@ -0,0 +1,18 @@
+import type { BaseLogger as PinoLogger } from 'pino';
+import { LogLevel } from '../logger';
+import { Attributes } from '../utils';
+import { LogWriter } from './common';
+
+export class PinoLogWriter implements LogWriter {
+  #pinoLogger: PinoLogger;
+  constructor(pinoLogger: PinoLogger) {
+    this.#pinoLogger = pinoLogger;
+  }
+  write(
+    level: LogLevel,
+    attrs: Attributes | null | undefined,
+    msg: string | null | undefined,
+  ): void {
+    this.#pinoLogger[level](attrs, msg || undefined);
```

Comment:
How does this integrate with Pino's child logic?

Reply:
Nope, all of the metadata and child mechanism is handled by Hive Logger (since you're already using it) and Pino and Winston (and all other writers) simply write.

Reply:
All of their writing mechanisms (like Pino and Winstons transports) will still work as expected.

Reply:
If it really doesn't matter to use their metadata and child features, then it is fine. I was concerning about if they have a special logic for their own child and metadata stuff.